internal/workflow_testsuite.go

```diff
@@ -539,6 +539,19 @@ func (e *TestWorkflowEnvironment) SetIdentity(identity string) *TestWorkflowEnvi
 	return e
 }
 
+// SetDetachedChildWait, if true, will make ExecuteWorkflow wait on all child
+// workflows to complete even if their close policy is set to abandon or request
+// cancel. If false, ExecuteWorkflow will wait on all children to complete
+// regardless of detached child. In situations with interminable cron-based
+// child workflows, setting this to false will allow the child to run once but
```

Comment:
Also the phrasing here is confusing, what does "allow" mean? Will it always wait for at least a single execution?

Reply:
Clarified.